go.mod

```diff
@@ -171,18 +168,53 @@ replace (
 
 require (
 	github.com/eclipse/che-go-jsonrpc v0.0.0-20200317130110-931966b891fe
-	github.com/gin-contrib/sse v0.0.0-20170109093832-22d885f9ecc7 // indirect
-	github.com/gin-gonic/gin v0.0.0-20180501062418-bd4f73af679e
-	github.com/gorilla/websocket v1.4.2
-	github.com/mattn/go-isatty v0.0.3 // indirect
+	github.com/gin-gonic/gin v1.8.2
+	github.com/gorilla/websocket v1.5.0
 	github.com/pkg/errors v0.9.1
-	github.com/sirupsen/logrus v1.4.2
-	github.com/stretchr/testify v1.6.1
-	github.com/ugorji/go v1.1.1 // indirect
-	gopkg.in/go-playground/assert.v1 v1.2.1 // indirect
-	gopkg.in/go-playground/validator.v8 v8.18.2 // indirect
+	github.com/sirupsen/logrus v1.9.0
+	github.com/stretchr/testify v1.8.1
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.21.0
-	k8s.io/apimachinery v0.21.0
-	k8s.io/client-go v0.21.0
+	k8s.io/api v0.26.1
+	k8s.io/apimachinery v0.26.1
+	k8s.io/client-go v1.5.2
```

Comment:
Also no-op upgrades, as they're fixed by the `replaces` section above :/

We should either actually update these packages (and update/remove the replaces for non-transitive dependencies) or leave these lines unchanged. Resetting these values to the original versions does not change anything in vendor except for modules.txt.